project/target/project-1/profile.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
I would not expect this to be under the target directory, as it is generated. This should probably be in the same location as the index.html (project/src/main/webapp) together with its associated images, script and css.